disco_aws_automation/disco_subnet.py

```diff
@@ -154,6 +155,15 @@ def create_peering_routes(self, peering_conn_id, cidr):
 
             self._route_table = self._find_route_table()
 
+    def delete_route(self, destination_cidr_block):
```

Comment:
Correct, the same destination CIDR can't be used more than once in the same route table.